doc/style-guide-markdown.md

```diff
@@ -0,0 +1,42 @@
+Style Guide for Markdown
+========================
+
+This extends the [general programming](style-guide-programming.md) portion of the [Style Guide](style-guide.md).
+
+
+* Markdown Formatting
+    * Follow GitHub Flavored Markup.
+    * Paragraphs should be all on one long line.
+    * Avoid HTML markup.
+    * Prefer to use indentation instead of code fences.
```

Comment:
This doesn't really answer my question.  I often have similar issues (which I can start to document if you'd like) with indents instead of code fences.  Why do you prefer indentation?